vuepress/docs/next/tutorials/devops/local-tips-and-tricks.md

```diff
@@ -0,0 +1,146 @@
+---
+sidebarDepth: 2
+--- 
+# Local Development Tips and Tricks
+Here you'll find a list of tips and tricks for local development environments. Please feel free to 
+provide your own or to discuss others on the [forum](https://forum.entando.org). 
+
+## Kubernetes
+Per [Getting Started](../../docs/getting-started/) we've recommended using Multipass as
+a quick way to spin up an Ubuntu VM and host a local Kubernetes cluster. Practically speaking most
+teams utilize a shared Kubernetes cluster, managed by an operations team and installed either on premise
+or your cloud provider of choice, for full integration testing, CI/CD, DevOps, etc. but there are many times
+when a local environment is useful. 
+
+## Windows development
+### Hyper-V IP changes
+**Q:** By design, Windows Hyper-V changes the IP address assigned to a VM (Multipass or other) when Windows 
+is restarted. How do I deal with this?
+
+**A:** The basic issue is that Windows Hyper-V makes it difficult to set
+a static IP for a VM. See this [forum post](https://techcommunity.microsoft.com/t5/windows-insider-program/hyper-v-default-switch-ip-address-range-change-ver-1809-build/m-p/261431) for details. 
+ 
+Locally the problem is that the ENTANDO_DEFAULT_ROUTING_SUFFIX (in your entando.yaml or similar) is based on the 
+VM IP address. That setting is used to generate ingress routes so it's not a simple fix to
+rewire those settings once Entando is installed. In production settings there's generally a dedicated network layer to manage IPs/routing 
+(both on premise and cloud) but those options are generally not readily available in a local setup. Here are a 
+few options short of modifying your router/switch settings: 
+
+#### Option 1: Reinstall Entando
+Okay, this is a terrible option since you already installed it once and don't want to do it again every 
+time Windows surprises you with an update. Nevertheless, it works so it's worth mentioning.
+
+#### Option 2: Manually update your hosts file
+This is the simplest option but a bit annoying to do after every restart. 
+You need two pieces of information for this workaround and you'll need administrator access to do this.
+
```

Comment:
The simplest option to re-enable external access to your cluster is to update your hosts file. Although you'll have to update your hosts file after each restart, it's a straightforward fix that doesn't require you to have experience configuring Hyper-V.

Reply:
Option 2 doesn't require Hyper-V knowledge. It's purely a Windows networking hack.